.travis.yml

```diff
@@ -1,3 +1,14 @@
 language: node_js
 node_js:
-  - "12"
\ No newline at end of file
+  - '12'
```

Comment:
Added this in too, but didn't think of commenting the tests out. So yes this should let us merge now, I think.